components/github-api-client/src/config.rs

```diff
@@ -14,16 +14,6 @@
 
 /// URL to GitHub API endpoint
 pub const DEFAULT_GITHUB_API_URL: &'static str = "https://api.github.com";
-/// URL to GitHub Token endpoint
-pub const DEFAULT_GITHUB_TOKEN_URL: &'static str = "https://github.com/login/oauth/access_token";
-/// Default Client ID providing a value in development environments only.
-///
-/// See https://developer.github.com/apps
-pub const DEV_GITHUB_CLIENT_ID: &'static str = "Iv1.732260b62f84db15";
-/// Default Client Secret providing a value in development environments only.
-///
-/// See https://developer.github.com/apps
-pub const DEV_GITHUB_CLIENT_SECRET: &'static str = "fc7654ed8c65ccfe014cd339a55e3538f935027a";
```

Comment:
I didn't realize we don't need any of this stuff for the GH API calls we're doing (for connecting plans to Builder).